.secretsignore

```diff
@@ -16,3 +16,4 @@ wJalrXUtnFEMI/K3MDENG/bPxRfiCYEXAMPLEKEY
 wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
 zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
 10a718b3f285d89c36e9864494cdd1507f3bc85b342df24736ea81f9a1134bcc
+-----BEGIN PRIVATE KEY-----
```

Comment:
is this intentional?

Reply:
Yes, this file includes an example with this line that fails commits